Dockerfile

```diff
@@ -18,6 +18,7 @@ COPY api/ api/
 COPY clients/ clients/
 COPY controller/ controller/
 COPY cloud/ cloud/
+COPY observability/ observability/
```

Comment:
looks like the e2e tests are failing because this dir is also missing from the Tiltfile Docker build step, could you add this there as well?

Reply:
Those are still now failing... Do you have any idea what might be wrong here?

Reply:
I think the issue is you don't have the `LINODE_TOKEN` secret in your fork? could you try adding that in?